python/lsst/validate/drp/report_performance.py

```diff
@@ -0,0 +1,158 @@
+# LSST Data Management System
+# Copyright 2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from astropy.table import Column, Table
+
+from lsst.validate.base import load_metrics
```

Comment:
O.K. Thanks.  The AP team are using `lsst.verify` for the `verify_ap` package.  I'm not sure they know that SQuaSH will not be able to consume that until Sept.  It might be worth touching base with Eric.